docs_user/modules/proc_performing-a-fast-forward-upgrade-on-compute-services.adoc

```diff
@@ -120,6 +120,12 @@ $ oc exec -it nova-cell1-conductor-0 -- nova-manage db online_data_migrations
 $ oc rsh nova-cell0-conductor-0 nova-manage cell_v2 discover_hosts --verbose
 ----
 
+. Verify if the existing test VM instance is running:
++
+---
+${BASH_ALIASES[openstack]} server --os-compute-api-version 2.48 show --diagnostics test 2>&1 || echo FAIL
+---
+
```

Comment:
We may want to add upstream-only conditionals here (in another PR perhaps because this is about more than just this edit). AFAIK the downstream docs won't have the concept of a test instance.